doc/api/webstreams.md

```diff
@@ -283,12 +283,11 @@ added: v16.5.0
   `ReadableStream`'s data will be written.
 * `options` {Object}
   * `preventAbort` {boolean} When `true`, errors in this `ReadableStream`
-    will not cause `transform.writable` to be aborted.
-  * `preventCancel` {boolean} When `true`, errors in the destination
-    `transform.writable` is not cause this `ReadableStream` to be
-    canceled.
+    will not cause `destination` to be aborted.
+  * `preventCancel` {boolean} When `true`, errors in the `destination`
+    is not cause this `ReadableStream` to be canceled.
```

Comment:
```suggestion
    will not cause this `ReadableStream` to be canceled.
```

Reply:
Thanks for the review, the fixed commit has been pushed.